examples/baseball.py

```diff
@@ -147,29 +143,47 @@ def get_site_stats(array, player_names):
     Return the summarized statistics for a given array corresponding
     to the values sampled for a latent or response site.
     """
+    # TODO: only use pandas (or any lightweight tabular package) to display final result
     if len(array.shape) == 1:
         df = pd.DataFrame(array).transpose()
     else:
         df = pd.DataFrame(array, columns=player_names).transpose()
     return df.apply(pd.Series.describe, axis=1)[["mean", "std", "25%", "50%", "75%"]]
 
 
-def summary(traces, sites, player_names, transforms={}):
+def summary(trace_posterior, sites, player_names, transforms={}, diagnostics=True):
     """
     Return summarized statistics for each of the ``sites`` in the
     traces corresponding to the approximate posterior.
     """
-    marginal = EmpiricalMarginal(traces, sites).get_samples_and_weights()[0].numpy()
+    marginal = EmpiricalMarginal(trace_posterior, sites).get_samples_and_weights()[0]
+
+    if diagnostics and trace_posterior.sampler.num_chains > 1:
```

Comment:
With `num_chains=1`, sampler will not have `num_chains` set, and this will throw an error.

Reply:
you are right, I'll change to trace_posterior.num_chains instead.